editor/blocks/quote/index.js

```diff
@@ -0,0 +1,42 @@
+const Editable = wp.blocks.Editable;
+const { html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: html( 'blockquote > p' ),
```

Comment:
>Could we join the output directly using hpq?

Not super cleanly, but possible. Since each matcher is just a function receiving a node, you can create your own pass-through:

```js
attributes: {
	value: ( node ) => query( 'p', html() )( node ).join()
}
```

Or some compose-y equivalent.

I don't know that it's necessary a big issue to need to deal with this in the render behavior though.